src/RA_Dlg_Memory.cpp

```diff
@@ -57,33 +57,24 @@ _NODISCARD _CONSTANT_FN GetMaxNibble(_In_ MemSize size) noexcept
     switch (size)
     {
         default:
-        case MemSize::EightBit:
-            return 1U;
-        case MemSize::SixteenBit:
-            return 3U;
-        case MemSize::ThirtyTwoBit:
-            return 7U;
+        case MemSize::EightBit: return 1U;
+        case MemSize::SixteenBit: return 3U;
+        case MemSize::ThirtyTwoBit: return 7U;
     }
 }
 
-
-INT_PTR CALLBACK MemoryViewerControl::s_MemoryDrawProc(HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
+LRESULT CALLBACK MemoryViewerControl::s_MemoryDrawProc(HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
 {
     switch (uMsg)
     {
         case WM_NCCREATE:
-        case WM_NCDESTROY:
-            return TRUE;
+        case WM_NCDESTROY: return TRUE;
```

Comment:
I'm not sure I like this formatting change. It's fine if all the cases collapse to a single line,  like above in `GetMaxNibble`, but interspersed with multi-line blocks, it feels unnatural. I think I'd prefer to side on making the simple switches more verbose rather than trying to collapse parts of a more complex switch.

Recommend changing `AllowShortCaseLabelsOnASingleLine` to false, since it seems to ignore the "Allow" and behaves like "EnforceShortCaseLabelsOnASingleLine".

Reply:
I did that to prevent changes to areas that already have them on one line. It can go to false. 

"Short" depends on the line length currently set, which is 120. So it might look weird sometimes. Putting it to false is what I prefer as well.